### ISSUE TITLE
Expected(?) bootloop after flashing Lineage OS 17.1 on Cactus

### ISSUE THREAD

**Issue by LouizFC** [xiaomi-mt6765/android_device_xiaomi_mt6765-common#6]
Greetings.

I have a Redmi 6a and I tried to flash the [this ROM](https://samarv-121.github.io/lineageos-17.1-cactus-cereus/). In the "Know Issues" section, there is the following issue:

> Bootloop on first install (must force reboot after clean install)

I found a bit unclear what I should do to fix this issue (I am not a native English speaker, so I probably can blame that).

I tried flashing the Stock ROM back and flashing Lineage again, but had the same result.

**So, how do I prevent the mentioned bootloop?**

Thanks in advance

**Comment by minhducsun2002** [comment]
> (must force reboot after clean install)

**Comment by LouizFC** [comment]
@minhducsun2002 thank you for your reply.

This "force reboot" is the part that I found unclear (I should have specified it, sorry).

I already tried holding the power button (only the power button if it helps) at the Mi logo, it turns off but the loop begins again. I also tried doing it at the lineage os logo, but the "loop" turns the phone off quickier than the "holding power button" powering off delay.

What I found unclear is how and when should I do this reboot. With that clear I can know if the behaviour that I am experiencing is a bug or if I am missing some crucial step
